### PR TITLE
Training example

### PR DESCRIPTION
**Pull request by d-monnet**

### CODE REVIEW

**Comment by d-monnet** [comment]
Hi @farhadrclass, could you add the callback in the DNNTraining file ? I'm not sure how to do that (I think FluxNLPModels-tools is required ?)

**Comment by farhadrclass** [comment]
@d-monnet  I see you comment but I can not see the comment ?

**Comment by d-monnet** [comment]
> @d-monnet I see you comment but I can not see the comment ?

You mean you cannot read it or you don't understand it ?

**Comment by farhadrclass** [comment]
I got an email that you commented, but when I check github, it doesn't show anything

**Comment by farhadrclass** [comment]
image

**Comment by d-monnet** [comment]
> image

It's a review I started, so maybe that's why you can't see it ? Anyway, here is the comment: could you add the callback in the DNNTraining file ? I'm not sure how to do that (I think FluxNLPModels-tools is required ?)

File: docs/src/DNNTraining.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    MaxPool((2, 2)),
+    Flux.flatten,
+    Dense(256 => 120, relu),
+    Dense(120 => 84, relu),
+    Dense(84 => 10),
+  ) |> device
+
+nlp = FluxNLPModel(model, train_loader, test_loader; loss_f = loss)
+
+# R2
```
Comment (farhadrclass) [inline docs/src/DNNTraining.md]: yes these callbacks are too simple , I will update this file today or tomorrow with more 
https://github.com/Farhad-phd/FluxNLPModel-tools.jl/blob/main/Experiments/MNIST/Lenet_MNIST_SR2.jl
For more info on the call back

File: docs/src/DNNTraining.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+# ## Report on train and test
+# train_acc = FluxNLPModels.accuracy(nlp; data_loader = train_loader)
+# test_acc = FluxNLPModels.accuracy(nlp) #on the test data
+
+#MultiPrecisionR2
+FP = [Float32, Float64] # define floating point formats used by the algorithm for objective and gradient evaluation
+mpmodel = FPMPNLPModel(nlp,FP); # instanciate a Floating Point Multi Precision NLPModel (FPMPNLPModel)
+stat = MPR2(mpmodel) # run the algorithm
+```
```
Comment (farhadrclass) [inline docs/src/DNNTraining.md:121]: what is the issue that we need to comment them out?

**Comment by farhadrclass** [comment]
> > image
> 
> It's a review I started, so maybe that's why you can't see it ? Anyway, here is the comment: could you add the callback in the DNNTraining file ? I'm not sure how to do that (I think FluxNLPModels-tools is required ?)

you cold use a simple call back that I shared in FluxNLPModels.jl example which basically update the batch or In the example I added in the comment of the code review 

I can take a look tommorrow if you want ?

**Pull request merged**